style.css

```diff
@@ -607,6 +602,183 @@ textarea::placeholder {
   color: #5e6c84;
 }
 
+/* Creat style for popup window */
+
+.modal-section {
+  position: fixed;
+  width: 100%;
+  min-height: 100%;
+  transition: 0.2s ease-in-out;
+  background-color: rgba(165, 185, 224, 0.5);
+  -webkit-backdrop-filter: blur(5px);
+  margin-bottom: 50px;
+  backdrop-filter: blur(5px);
+  display: none;
+  overflow: auto;
+  align-items: center;
+}
```

Comment:
- [ ] It's awesome the way you rendered your POP details dynamically but lets quickly address these few issues that has to do with the given template.

**For the desktop version,**
1. Kindly align the button elements properly with the tech stacks. I could see an horizontal element in the figma design but it's missing from your design. ℹ️ 
2. The whole pop container has a margin bottom. 👍 

## Figma Design
<img width="540" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225247367-167898a4-a2a2-4c2c-bef9-16766058e9ac.png">

## Your Design
<img width="748" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225247599-596ddf65-058e-429d-a028-d166d378318b.png">

**For mobile version,** 
1. Kindly ensure that all the elements are contained in the pop container.
2. When I clicked on the see project, the pop-up displayed different images.
3. The mobile version is totally different from what we have in the figma design.

## Figma Design
<img width="144" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225249641-8c118fe3-aa18-4c0b-8f7b-94861b22696b.png">

## Your Design
<img width="290" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/225249406-16937173-90c0-4115-ad5b-76ac15711450.png">